src/components/HitherJobMonitor/HitherJobMonitorControl.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FunctionComponent, useContext, useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { HitherContext, HitherJob } from '../../extensions/common/hither';
+import { getPathQuery } from '../../kachery';
+import { DocumentInfo } from '../../reducers/documentInfo';
+
+type Props = {
+    documentInfo: DocumentInfo,
+    websocketStatus: string
+}
+
+const HitherJobMonitorControl: FunctionComponent<Props> = ({ documentInfo, websocketStatus }) => {
+    const [hitherJobs, setHitherJobs] = useState<HitherJob[]>([])
```

Comment:
Same technique as in HitherJobMonitor

Reply:
Ah yes. Wondering if the function could be extracted. Would it be possible to make it part of the `hither` context object? In addition to avoiding the repeat code, that maybe could help mitigate any issues associated with the `hither` object somehow disappearing under the feet of this effect hook?